### PR TITLE
Fix squad pipeline

### PR DESCRIPTION
**Pull request by anmarques**
Enable the use of squad examples with multiple features so it matches the score computed during training.

### CODE REVIEW

File: src/deepsparse/transformers/pipelines/question_answering.py [hunk 1]
```diff
@@ -147,6 +147,32 @@ def __init__(
 
         super().__init__(**kwargs)
 
+    def __call__(self, *args, **kwargs) -> BaseModel:
```
Comment (bfineran) [inline src/deepsparse/transformers/pipelines/question_answering.py:150]: does this call override add something besides unpacking `engine_inputs, postprocess_kwargs` this should already be supported by the base class right?

**Review by bfineran — CHANGES_REQUESTED** [review]
looks like quality needs to be updated. LGTM overall, my main question here is - is this backwards compatible with the QA models trained before the latest transformers updates?

File: src/deepsparse/transformers/pipelines/question_answering.py (hunk 1 above)
Comment (anmarques) [reply src/deepsparse/transformers/pipelines/question_answering.py:150]: The main difference this override adds is on line 162 where it calls the engine multiple times if you have more than one set of inputs. The unpacking is just something I simplified for this specific case.

Comment (bfineran) [reply src/deepsparse/transformers/pipelines/question_answering.py:150]: ah. nice, missed that. is that to support more spans? would prefer to instead add an overridable method in the base Pipeline `def engine_forward(self, inputs, **kwargs)` that just does the engine forward, and then override in the QA pipeline. that way if there's changes to `__call__` in the future they will be propagated to QA

**Comment by anmarques** [comment]
I don't think it'll affect backwards compatibility. This is only about how to process the inputs/outputs.

**Comment by bfineran** [comment]
ok can you test this branch against one of the older squad models in the zoo? LGTM after that, pending the suggested __call__ refactor

**Review by mgoin — APPROVED** [review]
LGTM, I like this solution using engine_forward

File: src/deepsparse/transformers/pipelines/question_answering.py (hunk 1 above)
Comment (bfineran) [reply src/deepsparse/transformers/pipelines/question_answering.py]: implementation looks great!

**Pull request merged**